megamek/src/megamek/client/ui/swing/PlanetaryConditionsDialog.java

```diff
@@ -348,7 +368,10 @@ private void refreshConditions() {
     }
 
     private void setConditions() {
+            setConditions(true);
+    }
 
+    private void setConditions(boolean shouldSendToClient) {
```

Comment:
This has no effect on the way it works, but this doesn't actually send it to the client. It instructs the client to send it to the server.

Reply:
Changed it the parameter name to `shouldSendToServer`.